site/oauth2-examples.md

```diff
@@ -151,8 +152,18 @@ in `advanced.config`:
 
 ### <a id="identity-provider-initiated-logon" class="anchor" href="#identity-provider-initiated-logon">Identity-Provider initiated logon</a>
 
-When RabbitMQ is provided as a service from a web portal, it is easy to navigate to the RabbitMQ Management UI
-with a single click. The web portal retrieves a token before taking the user to the RabbitMQ Management UI web page.
+Alike Service-Provider initiated logon, with Idp-initiated logon users land to RabbitMQ management ui with a valid token.
+These two scenarios below are examples of Idp-initiated logon:
+
+* RabbitMQ is behind a web portal which conveniently allow users to navigate directly to RabbitMQ fully authenticated
+* There is an OAuth2 proxy in between users and RabbitMQ which intercepts their requests and forwards them to RabbitMQ injecting the token into the HTTP `Authorization` header  
+
+The latter scenario is demonstrated [here](oauth2-examples-proxy.html). The former scenario is covered in the following section.
+
+#### Idp-initiated logon via the login endpoint
+
+A web portal offers their authenticated users, the option to navigate to RabbitMQ by submitting a form with their OAuth  token in `access_token` form field as it is illustrated below:
+
```

Comment:
- remove the comma after users in the first sentence. 
  - I think there is an extra space after "OAuth" and "token" in the 2nd sentence - remove it
   - add the word "the" before `access_token`
   - Lets not say "As it is illustrated below" for accessibility users, how about. "...form field as provided in the following example: